frontend/src/store/index.ts

```diff
@@ -0,0 +1,9 @@
+import { configureStore } from "@reduxjs/toolkit";
+import userReducer from "./slicers/userSlice";
+export const store = configureStore({
+    reducer: {
+        user: userReducer
+    }
+})
+export type RootState = ReturnType<typeof store.getState>
+export type AppDispatch = typeof store.dispatch
```

Comment:
There is the code in the example:
```
export const useAppDispatch = () => useDispatch<AppDispatch>();
export const useAppSelector: TypedUseSelectorHook<RootState> = useSelector;
```
you can use it to simplify working with types